### ISSUE TITLE
no function clause matching in EEx.eval_string/3

### ISSUE THREAD

**Issue by mbuhot** [wingyplus/redoc_ui_plug#8]
Hello, thanks for publishing this package!

While trying it out I experienced an error: 

```
** (exit) an exception was raised:
    ** (FunctionClauseError) no function clause matching in EEx.eval_string/3
        (eex 1.14.0) lib/eex.ex:274: EEx.eval_string("<!doctype html>\n<html>\n  <head>\n    <title>ReDoc</title\n    <meta charset=\"utf-8\">\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n    <link href=\"https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700\" rel=\"stylesheet\">\n\n    <style>\n      body {\n        margin: 0;\n        padding: 0;\n      }\n    </style>\n  </head>\n  <body>\n    <redoc spec-url=\"<%= spec_url %>\"></redoc>\n    <script src=\"https://cdn.jsdelivr.net/npm/redoc@<%= Plug.HTML.html_escape(redoc_version) %>/bundles/redoc.standalone.js\"></script>\n  </body>\n</html>\n", %{redoc_version: "latest", spec_url: "/json_api/openapi"}, [])
        (redoc_ui_plug 0.2.0) lib/plug/redoc_ui.ex:60: Redoc.Plug.RedocUI.call/2
```

EEx.eval_string requires that the bindings parameter be a keyword list, not a map: https://github.com/elixir-lang/elixir/blob/v1.14/lib/eex/lib/eex.ex#L275

**Comment by wingyplus** [comment]
Fixed in #10

**Comment by wingyplus** [comment]
Released in [0.2.1](https://hexdocs.pm/redoc_ui_plug/0.2.1). 

Thank you for your report. I'm going to close the issue now.  🙇
